vscode-extension/src/devbox.ts

```diff
@@ -7,63 +7,77 @@ interface Message {
 }
 
 export async function devboxReopen() {
-    await window.withProgress({
-        location: ProgressLocation.Notification,
-        title: "Setting up your Devbox environment. Please don't close vscode.",
-        cancellable: true
-    },
-        async (progress, token) => {
-            token.onCancellationRequested(() => {
-                console.log("User canceled the long running operation");
-            });
 
-            const p = new Promise<void>(async (resolve, reject) => {
+    if (process.platform === 'win32') {
+        const seeDocs = 'See Devbox docs';
+        const result = await window.showErrorMessage(
+            'This feature is not supported on your platform. \
+            Please open VSCode from inside devbox shell in WSL using the CLI.', seeDocs
+        );
+        if (result === seeDocs) {
+            env.openExternal(Uri.parse('https://www.jetpack.io/devbox/docs/ide_configuration/vscode/#windows-setup'));
+        }
+    } else {
```

Comment:
Is it possible to do early return on line 18, and remove the `else` here for better readability?